src/objit/src/expr/ob_expr_info_flag.h

```diff
@@ -22,7 +22,7 @@ enum ObExprInfoFlag
 {
   IS_CONST = 0x0,   // const expression, e.g. 1.2, 'abc'
   IS_COLUMN,        // column expression, e.g. C1, T1.C1
-  IS_STATIC_PARAM,         // vaiable expression, e.g. ?
```

Comment:
error: patch failed: src/sql/resolver/dml/ob_dml_resolver.cpp
error: patch failed: src/sql/resolver/expr/ob_raw_expr_copier.cpp

remove the modification of two file, there is conflict with internal codebase

Reply:
remove conflicting code (https://github.com/oceanbase/oceanbase/pull/1546/commits/55cfa99aef1ed1b1e0a913b068c8a3cbddb91ce8)